packages/core-components/src/layout/TabbedCard/TabbedCard.tsx

```diff
@@ -61,6 +62,7 @@ const BoldHeader = withStyles(
 type Props = {
   /** @deprecated Use errorBoundaryProps instead */
   slackChannel?: string;
+  style?: CSSProperties;
```

Comment:
I would recommend using `className` instead as we could apply to multiple elements conceptually. Is there a particular reason to use style?